ScienceDirect.js

```diff
@@ -25,23 +25,24 @@ function detectWeb(doc, url) {
 		return false;
 	}
 
-	if ((url.indexOf("pdf") !== -1 &&
-			url.indexOf("_ob=ArticleURL") === -1 &&
-			url.indexOf("/article/") === -1) ||
+	if ((url.includes("pdf") &&
+			!url.includes("_ob=ArticleURL") &&
+			!url.includes("/article/")) ||
 		url.search(/\/(?:journal|bookseries|book|handbooks|referenceworks)\//) !== -1 ||
-		url.indexOf("_ob=ArticleListURL") !== -1) {
+		url.includes("_ob=ArticleListURL")) {
```

Comment:
Do you happen to know for what this last case block here is about? We don't have anymore an test url including `_ob=ArticleListURL`.

This was first introduced by the commit https://github.com/zotero/translators/commit/06047f8893d04b703ae0d692a1906acf2df793c1